topics/basics/editing.md

```diff
@@ -3,8 +3,9 @@
 <!-- Copyright 2000-2022 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
 
 * Code Completion
+* [](postfix_completion.md)
```

Comment:
include sub pages here as well

Reply:
Are you sure? Other sections like "Live Templates" and "File and Code Templates" do not include their subpages and I based on them. Let me know if I should add subsections for all listed pages.

Reply:
@karollewandowski Again, don't overdo it. If you think subpages would be nice, add them to the other sections as well. Otherwise, just go ahead.

Reply:
Leaving it then, it can be fixed in post-production together with the other sections.